src/DynamoCoreWpf/Views/Core/NoteView.xaml.cs

```diff
@@ -86,9 +88,7 @@ void OnViewModelRequestsSelection(object sender, EventArgs e)
 
         void OnNoteTextPreviewMouseDown(object sender, MouseButtonEventArgs e)
         {
-            System.Guid noteGuid = this.ViewModel.Model.GUID;
-            ViewModel.WorkspaceViewModel.DynamoViewModel.ExecuteCommand(
-                new DynCmd.SelectModelCommand(noteGuid, Keyboard.Modifiers.AsDynamoType()));
+            ViewModel.SelectNoteAndPinnedNode();
```

Comment:
If I remember correctly, all the command will be recorded into xml if Dynamo crashes so the code here does make a difference, but probably not matter too much since it is not likely crash would happen immediately after this particular action

Reply:
@QilongTang I can move the code on the `SelectNoteAndPinnedNode` so that it is on the command

Reply:
@QilongTang , ok just moved selection to command on PR: c6bf701